### PR TITLE
SQL Server - setting presume abort for XA in-doubt transaction

### PR DESCRIPTION
**Pull request by jjargot**
This is mandatory setting for all databases involved in a XA transaction: bonita engine database and business data database.

### CODE REVIEW

File: md/database-configuration.md [hunk 1]
```diff
@@ -389,9 +389,31 @@ ALTER DATABASE BONITA_BPM SET ALLOW_SNAPSHOT_ISOLATION ON
 ALTER DATABASE BONITA_BPM SET READ_COMMITTED_SNAPSHOT ON
 ALTER DATABASE BONITA_BPM SET MULTI_USER
 ```
-
 See [MSDN](https://msdn.microsoft.com/en-us/library/ms175095(v=sql.110).aspx).
 
+
+Run the script below to avoid the SQL Server change the status of the database to SUSPECT during database server startup when in-doubt XA transactions are found.
```
Comment (educhastenier) [inline md/database-configuration.md]: To visualize a Carriage Return character, add 2 spaces at the end of line : `  `

**Review by educhastenier — CHANGES_REQUESTED** [review]

**Comment by educhastenier** [comment]
retest this please

File: md/database-configuration.md [hunk 2]
```diff
@@ -389,9 +389,32 @@ ALTER DATABASE BONITA_BPM SET ALLOW_SNAPSHOT_ISOLATION ON
 ALTER DATABASE BONITA_BPM SET READ_COMMITTED_SNAPSHOT ON
 ALTER DATABASE BONITA_BPM SET MULTI_USER
 ```
-
 See [MSDN](https://msdn.microsoft.com/en-us/library/ms175095(v=sql.110).aspx).
 
+#### Recommended configuration in-doubt xact resolution
```
Comment (educhastenier) [inline md/database-configuration.md]: **for** in-doubt ...

File: md/database-configuration-for-business-data.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Configuration is done per tenant. If you have multiple tenants to configure, you
 
 ## Database creation
 
-In your RDBMS, create a database to be used to store business object. Do not reuse the Bonita BPM Engine database.
+In your RDBMS, create a database to be used to store business object. Do not reuse the Bonita BPM Engine database. Follow the instructions in the [Specific database configuration](database-configuration.md#Specific database configuration) section.
```
Comment (educhastenier) [inline md/database-configuration-for-business-data.md]: This link is not valid. It must point to an Anchor `<a id="specific_database_configuration">` that must exist, and must not contain space characters.

Comment (educhastenier) [reply md/database-configuration-for-business-data.md]: Have a look at https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md for how to use anchors

Comment (jjargot) [reply md/database-configuration-for-business-data.md]: I tried to mimic some other links that I found in doc

File: md/database-configuration.md (hunk 2 above)
Comment (jjargot) [reply md/database-configuration.md]: Do I need to do something ?

**Comment by jjargot** [comment]
done

File: md/database-configuration.md (hunk 2 above)
Comment (educhastenier) [reply md/database-configuration.md]: Add the word **for**

**Comment by educhastenier** [comment]
retest this please

**Comment by jjargot** [comment]
The word *for* has been added

File: md/database-configuration.md (hunk 2 above)
Comment (jjargot) [reply md/database-configuration.md]: added

**Comment by educhastenier** [comment]
Looks good to me

**Pull request merged**